CONTRIBUTING.md

```diff
@@ -34,16 +34,14 @@ When **requesting a feature**, context is key. Some of the questions we want to
 - How urgent is the need (nice to have feature or need to have)?
 - Does this align with the goals of Snap?
 
-When **proposing a new approach**, understanding the use case is key. We organize these requests into a RFC process. RFCs must include:
+When **proposing a RFC**, understanding the use case is key. This type of issue includes:
```

Comment:
`an RFC` sounds slightly more correct to me than `a RFC`, not sure which is correct though